java/flight/src/main/java/org/apache/arrow/flight/FlightClient.java

```diff
@@ -123,13 +126,16 @@ public FlightClient(BufferAllocator incomingAllocator, Location location) {
         .transform(CallOptions.wrapStub(blockingStub, options).doAction(action.toProtocol()), Result::new);
   }
 
+  /**
+   * Authenticates with a username an password.
```

Comment:
an -> and

Also maybe Authenticates -> Authenticate

Reply:
fixed in pending commit.